docs/source/schema/directives.md

```diff
@@ -1,40 +1,73 @@
 ---
-title: Using schema directives
-description: Using schema directives to transform schema types, fields, and arguments
+title: Directives
+sidebar_title: Directives
+description: Configure schema types, fields, and arguments
```

Comment:
So there are essentially two kinds of directives in GraphQL: executable directives that you put in your query/mutation (like `@skip`), and schema directives that you put in your schema (like `@deprecated`). I mean the spec doesn't literally differentiate between them but in practice most directives are declared to go on a set of executable locations or a set of schema locations.

Previously this document specifically talked about "schema directives" a lot. Though it then mentioned `@skip` and `@include` anyway.

Changing the document to be about directives in general does make sense to me, but I think at some point you should actually explain that some directives go on operations and some on schemas? Especially since at some point in the file you do start talking specifically about defining schema directives.  (ie, these docs don't explain how you'd write code to react to an executable directive, which perhaps may just be that we don't have a great pattern, I dunno.)

Reply:
Good points here. Revisiting.